composer.json

```diff
@@ -36,5 +36,11 @@
 				"cat $1 | ./node_modules/.bin/newspack-scripts commitlint"
 			]
 		}
+	},
+	"config": {
```

Comment:
Is this change required for this PR? If not, we should probably handle this separately.

Reply:
Yep, mistake on my end.